### DISCUSSION TITLE
LazyGit shows nothing all of a sudden

### DISCUSSION THREAD

**Discussion by jkbellew** [jesseduffield/lazygit#2024] [General]
I just started using LazyGit and I like, but today it just stopped working in neovim. I call it up (mapped to <leader>gg) and the window opens and then nothing. Just a blank window. I have not updated anything at all and other than mapping LazyGit to the above mapping there is nothing else. It did work in Neovim for about a week. So no idea why. 

Anyone else had a similar issue? Here is a screenshot: 
![Screenshot 2022-07-03 110141](https://user-images.githubusercontent.com/796886/177045602-c8f88d76-71b8-4f8b-94c0-b989c0064539.png)

**Comment by mark2185** [comment]
Does it work outside of `neovim`?

**Reply by jkbellew** [reply]
yes is does just not with the vim plugin

**Reply by mark2185** [reply]
And what plugin is in question? What exactly is your mapping invoking?

**Reply by jkbellew** [reply]
Plug 'kdheepak/lazygit.nvim' - LazyGIt is what I'm invoking.

**Reply by mark2185** [reply]
Have you thought about asking there? Since it's working outside of neovim, doesn't seem like a `lazygit` problem.

**Comment by SayarB** [comment]
I am having a similar issue as well

**Comment by jonasmedeiros** [comment]
I fixed it by reinstalling nvim

**Comment by hafiedh** [comment]
reinstalling nvim
